tools/compare-TaskRun-Pod-creationTimestamp.py

```diff
@@ -33,14 +33,30 @@ def load_file(fd):
     return data
 
 
+def generate_unique_name_mapping(name, namespace):
+    return "{namespace}.{name}".format(
+        namespace=namespace,
+        name=name
+    )
+
+
 def doit(args, status_data):
     logging.info(f"Processing {args.taskruns_list.name}")
     data_taskruns = {}
     for i in load_file(args.taskruns_list)["items"]:
         try:
-            tr_name = i["metadata"]["name"]
+            # TaskRun Names can be duplicate across multiples namespaces
+            # Combine namespace+taskrun name to generate unique name
+            tr_namespace = i["metadata"]["namespace"]
+            tr_name = generate_unique_name_mapping(
+                name=i["metadata"]["name"],
+                namespace=tr_namespace
+            )
```

Comment:
Maybe that helper function is not even needed:
```suggestion
            tr_name = f'{i["metadata"]["name"]}.{tr_namespace}'
```

Reply:
I'll change to f-strings and remove the function if these changes won't repeat. Kept it initially to consider if we need to change this logic. I'll check it out.

Reply:
I've replaced it with f-strings now :)